index.html

```diff
@@ -134,7 +145,7 @@ <h2>About me</h2>
           of my work and experience! If you like what you see and have a project
           you need coded, don’t hestiate to contact me.
         </p>
-        <button>Get My Resume</button>
+        <button class="btn-general">Get My Resume</button>
```

Comment:
- [ ] This button is looking fine on the mobile version. I think the padding is a bit small on the desktop version. Kindly increase the left and right padding on the button as suggested by the previous reviewer. 🥸️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/206549032-0052f928-e153-4faf-9ee8-dc8265fdf83e.png" width="200">|<img src="https://user-images.githubusercontent.com/17243325/206549131-103f25dd-4bdf-4224-b276-d0a603708b04.png" width="200">|